app/views/layouts/application.html.erb

```diff
@@ -45,7 +45,7 @@
           <nav class="header__nav-links">
             <% if signed_in? %>
               <a href="<%= profile_path(current_user.display_id) %>" class="header__nav-link mobile__header__nav-link">
-                <%= truncate current_user.name %>
+                <span><%= truncate current_user.name %></span>
```

Comment:
Truncate uses the default of 30 characters, perhaps we should use something smaller like 15 characters.